content/en/integrations/faq/powershell-command-to-install-azure-datadog-extension.md

```diff
@@ -8,16 +8,82 @@ Datadog provides an Azure extension to assist with Agent deployment on Azure ins
 * [Introducing Azure monitoring with one-click Datadog deployment][1]
 * [Azure integration documentation][2]
 
-An alternative to the GUI install is via Powershell.
-To run the Datadog Agent in your Azure instances as an extension, use the following syntax:
+An alternative to the GUI install is via command line.
+To run the Datadog Agent in your Azure instances as an extension, use the following syntaxes:
+
+{{< tabs >}}
+{{% tab "Windows" %}}
+
+```powershell
+Set-AzureVMExtension -Name "DatadogAgent" -Publisher "Datadog.Agent" -Type "DatadogWindowsAgent" -TypeHandlerVersion "5.0" -Settings @{"site" = "datadoghq.com"; "agentVersion" = "latest"} -ProtectedSettings @{"api_key" = "$(DatadogApiKey)"} -DisableAutoUpgradeMinorVersion
+```
+More information on the syntax to set Azure instance extensions can be found in the (Azure Extension Set-AzureVMExtension documentation)[3].
+
+The Azure Extension supports two type of settings: normal settings and protected settings.
+The normal settings include:
+
+| Variable | Type | Description  |
+|----------|------|--------------|
+| `site` | String | Set the Datadog intake site, for example: `SITE=`{{< region-param key="dd_site" code="true">}} |
+| `agentVersion` | String | The Agent version to install, following the format x.y.z or latest |
+| `agentConfiguration` | URI | (optional) Url to the Azure blob contaning the Agent configuration as a zip. |
+| `agentConfigurationChecksum` | String | The SHA256 checksum of the Agent configuration zip file, mandatory if `agentConfiguration` is specified. |
+
+The protected settings are encrypted and only decrypted on the target virtual machine. The protected configuration is useful when the execution command includes secrets such as the `api_key`.
+
+| Variable | Type | Description  |
+|----------|------|--------------|
+| `api_key`| String | Adds the Datadog API KEY to the configuration file. |
+
+**Note**: If `agentConfiguration` and `api_key` are specified at the same time, the API key found in the `agentConfiguration` takes precedence. Also note that for now, if an API key is set on the target machine, it's not possible to change it with `Set-AzureVMExtension`.
+
+### Specifying a configuration URI
+This example shows how to specify a configuration for the Datadog Agent to use.
+The Datadog Agent configuration URI must be an Azure blob storage URI.
+The Datadog Windows Agent Azure Extension will check that the `agentConfiguration` URI comes from the `.blob.core.windows.net` domain.
```

Comment:
the code checks for `*.core.windows.net` but not specifically for the `blob` subdomain
https://github.com/DataDog/azure-extension/blob/master/windows-package/scripts/install.ps1#L231